src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java

```diff
@@ -36,8 +42,11 @@ public void EditPreferences_001_selectVE() {
     editPrefPage.selectPreferredEditor(VE);
     PreferencesPageObject prefPage = editPrefPage.clickSaveButton();
 
-    Assertion.assertTrue(prefPage.getBannerNotifications().isNotificationMessageVisible(),
-                         "Notification message is not visible");
+    List<Notification> confirmNotifications = prefPage.getNotifications(NotificationType.CONFIRM);
+    Assertion.assertEquals(confirmNotifications.size(),1,
+            "Number of action confirming notifications is invalid");
+    Assertion.assertTrue(confirmNotifications.stream().findFirst().get().isVisible(),
+            "Banner notification message is not visible");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Banner notification message is not visible" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)